### DISCUSSION TITLE
Feature Request: Resolving review comments

### DISCUSSION THREAD

**Discussion by TonCherAmi** [community/community#28] [Mobile] [answered]
It’s rather difficult to do reviews through the app without being able to resolve comments.

**Comment by candyho** [comment]
Hey @TonCherAmi , thanks for the suggestion. I can definitely understand why this would be handy. I’ll surface this to the team and add this to our improvement backlog.

**Reply by gaurav5430** [reply]
looking for the same thing, can't merge PRs from the mobile app, because can't resolve review comments from the app.

any chances this has been planned?

**Comment by brianlovin** [comment, marked as answer]
Hey @TonCherAmi and @eguevara – stand by, we are beta testing this functionality right now, and hoping to ship it to both iOS and Android by the end of the month.

**Comment by Sandesh9892** [comment]
When I try to verify for the student on Git Hub for accessing the student package in copilot, it always says that I am not verified even when I tried every document, card, or student card that the university gave me.
What do you think I should do?
